fabtests/pytest/efa/test_rma_bw.py

```diff
@@ -4,18 +4,16 @@
 @pytest.mark.parametrize("iteration_type",
                          [pytest.param("short", marks=pytest.mark.short),
                           pytest.param("standard", marks=pytest.mark.standard)])
-def test_rma_bw(cmdline_args, iteration_type, operation_type, completion_type):
-    from common import ClientServerTest
+def test_rma_bw(cmdline_args, iteration_type, operation_type, completion_type, memory_type):
+    from test_rdm import run_rdm_test
```

Comment:
Can we rename `run_rdm_test` to a more generalized name for its usage? It is a bit weird to see a `test_rma_bw` to call a `run_rdm_test` because they are different tests in fabtests.

Reply:
It's probably better to keep importing `ClientServerTest` and duplicate the body of `run_rdm_test` here, which is just three statements.